django_elasticsearch_dsl/search.py

```diff
@@ -13,12 +13,17 @@ def _clone(self):
         s._model = self._model
         return s
 
-    def to_queryset(self, keep_order=True):
+    def filter_queryset(self, queryset, keep_search_order=True):
```

Comment:
This should be a internal function so it should start with `_`

Reply:
I actually meant it to be public, so I could do something like this:
```py
class SearchQuerySet(QuerySet):
    def from_elastic(self, search):
        return search.filter_queryset(self)
```

Could you explain your reasoning?

Reply:
Interesting. I did not think about it before. Let me think about it so I can suggest a better approach.

Reply:
I think `to_queryset` should accept a `queryset` as optional parameter. So if `queryset` is passed, it will use that queryset. otherwise call the `get_queryset` method to get the `queryset`

@laevilgenius what do you think?

Reply:
Should we keep `to_queryset` backwards compatible? I'm thinking `def to_queryset(self, qs=None, keep_order=True)` vs `def to_queryset(self, keep_order=True, qs=None)`.